samples/crypto/ecjpake/sample.yaml

```diff
@@ -0,0 +1,18 @@
+sample:
+  description: This app provides an example of EC J-PAKE
+  name: EC J-PAKE example
+tests:
+  sample.ecjpake:
+    tags: introduction psa
+    platform_allow: >
+      nrf5340dk_nrf5340_cpuapp_ns nrf5340dk_nrf5340_cpuapp
+      nrf9160dk_nrf9160_ns nrf9160dk_nrf9160 nrf52840dk_nrf52840
+    harness: console
+    harness_config:
+      type: multi_line
+      regex:
+        - ".*Example finished successfully!.*"
+    integration_platforms:
+      - nrf5340dk_nrf5340_cpuapp
+      - nrf9160dk_nrf9160
+      - nrf52840dk_nrf52840
```

Comment:
add the _ns variants to integration platform?
And the - nrf52840dk_nrf52840, we should be consistent in the crypto samples.

@torsteingrindvik Any objections?

Reply:
My understanding from @vili-nordic  is that _ns platforms are not supported.

Reply:
tf-m does not support this yet...

Reply:
> tf-m does not support this yet...

Should _ns platforms be removed from `platform_allow` then too?

Reply:
probably. yes :)